charts/pega/values.yaml

```diff
@@ -219,3 +219,11 @@ installer:
     upgradeType: "in-place"
     # Specify target rules schema for migration and upgrade
     targetRulesSchema: ""
+
+# Hazelcast settings
```

Comment:
Are these settings needed both here and in the hazelcast values.yaml?

Reply:
yes, Few mandatory parameters of subchart values.yaml can be overridden in main chart values.yaml . The same technique has been applied to installer and pega-search subchart as well.